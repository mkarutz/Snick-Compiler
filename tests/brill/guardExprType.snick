proc main ()
    int x;

    x := 2;

    if x then
        write x;
    fi
end
