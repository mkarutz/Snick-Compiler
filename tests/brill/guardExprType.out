Failure: Guard expression in if and while statements must have type bool.
Compilation Failed
